playground/infrastructure/helper.py

```diff
@@ -35,37 +44,164 @@ def find_examples(work_dir: str) -> List[Example]:
     """ Find and return beam examples.
 
     Search throws all child files of work_dir directory files with beam tag:
-    /*
     Beam-playground:
         name: NameOfExample
         description: Description of NameOfExample.
         multifile: false
         categories:
             - category-1
             - category-2
-    */
 
     Args:
         work_dir: directory where to search examples.
 
     Returns:
         List of Examples.
     """
-    examples = [Example("", "", SDK_UNSPECIFIED, "", "", "", STATUS_UNSPECIFIED)]
+    examples = []
+    for root, _, files in os.walk(work_dir):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            if _match_pattern(filepath):
+                examples.append(_get_example(filepath, filename))
     return examples
 
 
 def get_statuses(examples: [Example]):
-    """ Receive statuses for examples and update example.status and example.pipelineId
+    """ Receive statuses for examples and update example.status and example.pipeline_id
 
     Use client to send requests to the backend:
     1. Start code processing.
     2. Ping the backend while status is STATUS_VALIDATING/STATUS_PREPARING/STATUS_COMPILING/STATUS_EXECUTING
-    Update example.pipelineId with resulting pipelineId.
+    Update example.pipeline_id with resulting pipelineId.
     Update example.status with resulting status.
 
     Args:
         examples: beam examples for processing and updating statuses.
     """
     # TODO [BEAM-13267] Implement
     pass
+
+
+def get_tag(filepath):
+    """Parse file by filepath and find beam tag (after the licence part, before the import part)
+
+    If file contains tag, returns tag as a map.
+    If file contains tag but tag has incorrect format, raise error
+    If file doesn't contain tag, returns None
```

Comment:
Shouldn't it be in the Returns section of docstring?

Reply:
Changed.